src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala

```diff
@@ -43,4 +44,5 @@ object ErrorCode {
   val MULTI_VALUE_ERROR = "MULTI_VALUE_ERROR"
   val UNDEFINED_VALUE_ERROR = "UNDEFINED_VALUE_ERROR"
   val REQUIRED_PROPERTY_IS_EMPTY = "REQUIRED_PROPERTY_IS_EMPTY"
+  val TOO_LONG_INPUT_ERROR = "TOO_LONG_INPUT_ERROR"
```

Comment:
Could this error be renamed to something like this: `MAX_CHARACTER_LIMIT_INPUT_ERROR`?

Reply:
Yeah, that looks good to me as well